### DISCUSSION TITLE
How to insert image when using template format

### DISCUSSION THREAD

**Discussion by trevvbai** [ClosedXML/ClosedXML#1956] [Q&A]
Hello! 

I am attempting to populate a template and need to insert an image into one of the cells within the template. I have not been able to find an example of this or see a similar question so far.

Could somebody provide a resource showcasing this? 

Thanks!

**Comment by deepk-kr** [comment]
var cell = FindCell(worksheet.Name, imagePlaceHolder);
            
            IXLPicture image;
            if (stepDown > 0)
            {
                cell = cell.CellBelow(stepDown);
                image = worksheet.AddPicture(path)
                     .MoveTo(cell);

            }
            else
            {
                image = worksheet.AddPicture(path)
                    .MoveTo(cell);
            }

//imagePlaceHolder is some text on sheet, FindCell -> worksheet.Search(imagePlaceHolder)
